packages/realm-web-integration-tests/src/utils.ts

```diff
@@ -34,3 +36,9 @@ export function createApp<FunctionsFactoryType extends object>() {
         baseUrl: BASE_URL,
     });
 }
+
+export function clearStorage() {
+    if (window.localStorage) {
```

Comment:
in other placed `window` is also checked.

Reply:
Agree - this was a bug. It's removed now.